alchy/model.py

```diff
@@ -126,12 +125,7 @@ class User(Model):
     @declared_attr
     def __table_args__(cls):  # pylint: disable=no-self-argument
         # pylint: disable=no-member
-        args, kargs = merge_declarative_args(cls.__bases__, '__table_args__')
```

Comment:
I've updated the code to include ``__local_table/mapper_args__`` from ``cls.mro()``, and ``__table/mapper_args__`` from mixins in ``cls.__bases__``.